src/app/organizations/collection/collection.component.html

```diff
@@ -31,6 +31,9 @@
 <app-loading [loading]="(loadingCollection$ | async) || (loadingOrganization$ | async)">
   <div *ngIf="organization$ | async as organization">
     <div class="container mb-5" *ngIf="collection$ | async as collection" fxLayout="column">
+      <div *ngIf="Dockstore.FEATURES.enableNotebooks" class="mt-3">
```

Comment:
Looking at the collection page in your video, it's not clear what the banner is referring. Perhaps we should mention the feature in the banner, like Charles suggested during the demo.

Reply:
Noted, saw your comment on the original Jira ticket and while doing those updates will add in a `feature` parameter